core/src/main/java/bio/terra/pearl/core/service/kit/StubPepperDSMClient.java

```diff
@@ -40,19 +44,21 @@ public String sendKitRequest(String studyShortcode, Enrollee enrollee, KitReques
 
     @Override
     public PepperKitStatus fetchKitStatus(UUID kitRequestId) {
+        logger.info("STUB fethcing kit status");
         return PepperKitStatus.builder()
                 .juniperKitId(kitRequestId.toString())
-                .currentStatus("SHIPPED")
+                .currentStatus("SENT")
```

Comment:
Should SENT be defined in a constant somewhere (for confidently changing the as we do here)?

Reply:
good thought.  Here the relevant enum is PepperKitStatus, and this test is testing that we can use both upper and lowercase versions of the value, so we're not using the enum itself